docs/tutorials/update-local-data-to-silo-storage-account.md

```diff
@@ -0,0 +1,68 @@
+The goal of this tutorial is to show how to upload the contents of a local folder to a silo storage account. We will be using a CLI job to do the upload. The job will run on the silo compute which, unlike the data scientist, does have access to the silo storage account.
+
+## Prerequisites
+To enjoy this tutorial, you will need:
+- a Federated Learning setup (provisioned using the templates in our [quickstart](../quickstart.md), for instance);
+- the [Azure ML CLI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-cli?tabs=public) installed on your machine;
+- some local folder with data to upload.
+
+## Procedure
+We will be using the Azure ML CLI to submit a job to the silo compute. The job will upload the local data to the silo storage account. The job is defined in a YAML file that we will call the "job YAML" and that you mill need to adjust to your silo. Detailed steps are as follows.
+
+### 0. Clone the repository
+First of all, you will want to clone the current repository to your machine, if you haven't already done so.
+
+### 1. Adjust the job YAML
+A sample job YAML is given [here](../../examples/cli-jobs/upload-local-data-to-silo-storage.yml) in our repository, in the `\examples\cli-jobs\upload-local-data-to-silo-storage.yml` file. The contents are shown below.
+```yaml
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+
+command: |
+  cp -r ${{inputs.local_data_folder}} ${{outputs.destination_folder}}
+inputs:
+  local_data_folder:
+    type: uri_folder
+    path: /path/to/local/data/folder/ # replace '/path/to/local/data/folder/' by the actual path to the folder whose contents you want to upload
+
+outputs:
+  destination_folder:
+    type: uri_folder
+    mode: upload
+    path: azureml://datastores/<your-silo-datastore-name>/paths/<your-custom-local-path>/ # replace '<your-silo-datastore-name>' by the actual datastore name for your silo, and <your-custom-local-path> by the path you want to use in the silo storage account
+
+environment: azureml:AzureML-sklearn-1.0-ubuntu20.04-py38-cpu@latest
+
+compute: azureml:<your-silo-compute-name> # replace '<your-silo-compute-name>' by the actual compute name for your silo
+```
+
+
+ You will need to adjust the following parameters.
+ - `inputs.local_data_folder.path`: the path to the local folder whose contents you want to upload.
+ - `compute`: the name of the silo compute. This is the compute that will run the job. It needs to match the datastore mentioned below.
+- `outputs.destination_folder.path`: the path in the silo storage account where you want to upload the local data. **The key part to adjust is the name of the datastore.** You also have the ability to adjust the detailed path in the storage account, if you so wish.
+  - :warning: Note: the files and folders in `inputs.local_data_folder.path` will be uploaded to the storage account at the `outputs.destination_folder.path`, **under a folder named `INPUT_local_data_folder`**.
```

Comment:
really? can't we fix that? maybe by using cp -r ${...}/* ?

Reply:
Done, even though I'm not sure why it worked :D